src/configurePipeline/services/azureDevOpsService.ts

```diff
@@ -0,0 +1,363 @@
+import * as Mustache from 'mustache';
+import * as uuidv1 from 'uuid/v1';
+import * as vscode from 'vscode';
+
+import { ServiceClientCredentials, ServiceClient, UrlBasedRequestPrepareOptions } from 'ms-rest';
+
+import { GitRepoDetails, WizardInputs } from '../model/Common';
+import { Constants } from '../constants';
+
+export class AzureDevOpsService {
+	private serviceClient: ServiceClient;
+	private organizationName: string;
+	private projectName: string;
+
+	public constructor(credentials: ServiceClientCredentials) {
+		this.serviceClient = new ServiceClient(credentials);
+	}
+
+	public async getRepositoryDetails(repositoryName: string, organizationName?: string, projectName?: string): Promise<any> {
```

Comment:
should we take org and project name as parameters ? It should use the class property get methods